packs/agents-of-edgewatch-bestiary/shikwashim-mercenary.json

```diff
@@ -889,7 +889,30 @@
                     "remaster": false,
                     "title": ""
                 },
-                "rules": [],
+                "rules": [
+                    {
+                        "domain": "all",
+                        "key": "RollOption",
+                        "option": "terrain-advantage",
+                        "toggleable": true
+                    },
+                    {
+                        "key": "EphemeralEffect",
+                        "predicate": [
+                            "terrain-advantage",
```

Comment:
Should we add `terrain:difficult` and `terrain:water` to the predicates for those who use Scene Regions?

Reply:
In my testing, it doesn't appear that target terrain is in the roll options.

Reply:
Nor is `target:speed:swim` exposed.